Error: Cannot find module 'babel/polyfill'

Hello,

Babel is actually required to run eslint-nibble, so it fails if Babel is not installed with this error :
```js
module.js:338
    throw err;
          ^
Error: Cannot find module 'babel/polyfill'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/.../my-project/node_modules/eslint-nibble/index.js:4:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

--- a/index.js
+++ b/index.js
@@ -1,5 +1,3 @@
 #!/usr/bin/env node
 
-// require('babel/register');
-require("babel/polyfill");
 module.exports = require('./lib/main');

